assets/src/edit-story/app/media/media3p/api/apiFetcher.js

```diff
@@ -100,17 +118,64 @@ class ApiFetcher {
       ['page_token', pageToken],
       ['key', API_KEY],
     ].filter((entry) => Boolean(entry[1]));
+
+    // eslint-disable-next-line no-return-await
+    return await this.fetch({ params, path: Paths.LIST_MEDIA });
+  }
+
+  /**
+   * Perform a GET request to the Media3P API to list categories.
+   * If the parameters are invalid or the server does not return a 200 response,
+   * an error is thrown.
+   *
+   * @param {Object} obj - An object with the options for the request.
+   * @param {?string} obj.filter  Filter details for items returned.
+   * Filter fields available:
+   * > provider - The media provider to query, or a universal list/search if
+   * left blank.
+   * @param {?string} obj.orderBy The sort order for returned results.
+   * Valid sort fields are:
+   * > 'trending' - The default. This is the categories that are trending now.
+   * @param {?number} obj.pageSize Maximum number of results to be returned by the
+   * server. If unspecified or zero, at most 20 media resources will be returned.
+   * @return {Promise<Object>} The response from the API.
+   */
+  async listCategories({
+    filter = null,
+    orderBy = null,
+    pageSize = null,
+  } = {}) {
+    validateCategoriesOrderBy(orderBy);
+    validatePageSize(pageSize);
+
+    const params = [
+      ['filter', filter],
+      ['order_by', orderBy],
+      ['page_size', pageSize],
+      ['key', API_KEY],
+    ].filter((entry) => Boolean(entry[1]));
+
+    // eslint-disable-next-line no-return-await
+    return await this.fetch({
```

Comment:
Is the await needed here, maybe the 'no-return-await' doesn't need to be suppressed?